King_of_my_Heart.tex

```diff
@@ -0,0 +1,64 @@
+\documentclass{leadsheet}
+
+\begin{document}
+
+\begin{song}[transpose={+2}]{title={King of my Heart},composer={John Mark McMillan, Sarah McMillan},key={G},tempo={68}}
+
+\begin{schedule}
+I -- S1 -- R -- S2 -- R -- B1 -- R -- ZS -- R -- B2 -- R
+\end{schedule}
+
+\begin{intro}
+||: ^{Em}\_\_ ^{D}\_\_ | ^{C}\_\_ ^{G}\_\_ :||
+\end{intro}
+
+\begin{verse}
+Let the | ^GKing of my heart be the | ^Cmountain where I ^Grun \\
+The | ^{Em}fountain I drink ^{D}from oh He | ^{C}is my ^{G}song \\
+Let the | ^{G}King of my heart be the | ^{C}shadow where I ^{G}hide \\
+The | ^{Em}ransom for my ^{D}life oh He | ^{C}is my ^{G}song
+\end{verse}
+
+\begin{chorus}
+You are | ^{Em}good ^{D}good | ^{C}oh ^{G}\_\_ \\
+You are | ^{Em}good ^{D}good | ^{C}oh ^{G}\_\_ \\
+You are | ^{Em}good ^{D}good | ^{C}oh ^{G}\_\_ \\
+You are | ^{Em}good ^{D}good | ^{C}oh ^{G}\_\_ \\
+\end{chorus}
+
+\begin{verse}
+Let the | ^GKing of my heart be the | ^Cwind inside my ^Gsails \\
+The | ^{Em}anchor in the ^Dwaves oh He | ^Cis my ^Gsong \\
+Let the | ^GKing of my heart be the | ^Cfire inside my ^Gveins \\
+The | ^{Em}echo of my ^Ddays oh He | ^Cis my ^Gsong
+\end{verse}
+
+\begin{bridge}[numbered=true]
+^{G}\_ You're never gonna let | never gonna let me down | \\
+^{Em}\_ You're never gonna let | never gonna let me down | \\
+^{G}\_ You're never gonna let | never gonna let me down | \\
+\textsuperscript{1st Ending} ^{Em}\_ You're never gonna let | ^{D}never gonna let me down | \textsuperscript{To Bridge 1}\\
+\textsuperscript{2nd Ending} ^{Em}\_ You're ^Dnever gonna let | ^{D}never gonna let me ^Ddown | \\
+\end{bridge}
+
+\begin{bridge}
+^{Em}\_ You're ^{D}never gonna let | ^{C}never gonna let ^{G}me down | \\
```

Comment:
Didn't we simplify the bridge's chords?

Reply:
Has been fixed with new bridges and Schedule (reference: https://youtu.be/EpqSbKYxd9Y)